src/components/chip-set/partial-styles/_file-picker.scss

```diff
@@ -0,0 +1,57 @@
+:host {
+    --dropzone-margin: 0.25rem;
+}
+
+:host(.is-file-picker.shows-dropzone) {
+    .mdc-text-field:not(.has-chips) {
+        .mdc-notched-outline {
+            &:before,
+            &:after {
+                box-sizing: border-box;
+                position: absolute;
+                margin: auto;
+                top: var(--dropzone-margin);
+                right: var(--dropzone-margin);
+                bottom: var(--dropzone-margin);
+            }
+
+            &:before {
+                // dropzone tip
+                content: attr(dropzone-tip);
+
+                display: inline-flex;
+                align-items: center;
+                justify-content: flex-end;
+
+                left: 40%; // prevents overlapping of `label` and tip text
```

Comment:
We might need to limit the max-width of the label too:

![image](https://user-images.githubusercontent.com/154725/124556700-e10a1000-de38-11eb-8421-b97dbce82551.png)

Reply:
this commit f7ba8c7c207815b4b3843e5fc96ffb37b8ce9b61 (fix(chip-set): make long labels truncate correctly when there is leading icon) was needed to make this work. The fixup after takes care of label width